assets/sass/cds/_sections.scss

```diff
@@ -410,7 +410,7 @@ span.livefr {
     }
   }
 
-  h5, p, a {
+  p, a {
```

Comment:
Looking through the html, it looks like the text elements the h5s used to wrap ("Partners", "Contact" etc.) are not wrapped in a p, and the text content is just loose in the div. The visible difference looks minor, but there does look to be a slightly different spacing and text size in this PR compared to the live site.

Reply:
done.

There is still a very slight spacing difference, but it's close enough that I don't think it matters at this point. Thoughts?